lib/sensors/led_driver.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+int kDefaultLedDriverAddress = 0x30;
+
+namespace hyped::sensors {
+class LedDriver {
+ public:
+  static std::optional<LedDriver> create(core::ILogger &logger,
+                                         std::shared_ptr<io::II2c> i2c,
+                                         const std::uint8_t device_address
+                                         = kDefaultLedDriverAddress);
```

Comment:
we don't use default parameters, but here we don't need the param at all so it's a kinda meaningless point to make